website/source/downloads_tools.html.erb

```diff
@@ -99,6 +99,9 @@ description: |-
       variable expansion, interpolation, inheritance with overrides and ability to update multiple consul servers.
       Reduces cost of maintaining larger configuration sets between environments by reducing restatement and manual editing of similar or predictably changing config properties.  MIT license, Written in GO.
   </li>
+  <li>
+    <a href="https://docs.flightpath.xyz/">Flightpath</a> - An xDS server that can configure Envoy to act as an Edge proxy for Consul Connect enabled services
```

Comment:
```suggestion
    <a href="https://docs.flightpath.xyz/">Flightpath</a> - A xDS server that can configure Envoy to act as an Edge proxy for Consul Connect enabled services
```

Reply:
This is certainly a tricky one.
I had the classic Tom and Jerry childhood and I've always seen "An MGM Cartoon" or "A Metro Goldwyn Mayer Cartoon". I think it is `An` if the following acronym start with a vowel sound.

But english is not my native language so I'm not too sure. Thanks for the suggestion, I've committed the changes.

Reply:
WOW, I learned something new! Turns out you were right, at least https://english.stackexchange.com/questions/8181/does-one-use-a-or-an-before-the-word-x-ray says so. Sorry for the back and forth, `An` is correct.

Reply:
I am not an english speaker myself.

Reply:
No worries! Didn't expect a stackexchange question on this 😛 
Shall I change it back to `An`?

Reply:
Yes please! 🙏

Reply:
@i0rek updated. Thanks for the review.